<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportComponent</class>
 <widget class="QWidget" name="ExportComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>88</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="publishToLabel">
       <property name="text">
        <string>Publish to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="assetTaskLabel">
       <property name="text">
        <string>Task</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="assetNameLabel">
       <property name="text">
        <string>AssetName:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="publishToTaskLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="AssetTaskComboBox">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="AssetNameLineEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
